XO/XO.py

```diff
@@ -0,0 +1,117 @@
+from typing import List, Union, Tuple
+import random
+
+N = 3
+EMPTY = 'E'
+PLAYERS_SIGNS = ['X', 'O']
+
+
+def get_names() -> List[str]:
+    inputs = []
+    for i in range(2):
+        inputs.append(input(f'Insert the name of player {i}\n'))
+    return inputs
+
+
+def get_action(current_player: str, board: List[list]) -> Tuple[int, int]:
+    """
+    Talks with the user and get an input which represents his move, verify that the move is valid.
+
+    :param current_player: The name of the player which is playing now
+    :param board: The board game before the player did his move
+    :return: A tuple with the new position coordinates
+    """
+    error_message = 'Error'
+    texts = (f'{current_player}, insert line number (0/1/2)\n', f'{current_player}, insert column number (0/1/2)\n')
```

Comment:
This is still a bit confusing to read..
Better to just get the two inputs, as there's no need for generalization here:
```
row = f'{current_player}, insert line number (0/1/2)\n'
column = f'{current_player}, insert column number (0/1/2)\n'
```